keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandlerTests.kt

```diff
@@ -383,15 +382,15 @@ internal class SecurityGroupHandlerTests : JUnit5Minutests {
       setupVpc()
     }
 
-    sequenceOf(
+    mapOf(
       "create" to SecurityGroupHandler::create,
       "update" to SecurityGroupHandler::update
     )
-      .forEach { (methodName, handlerMethod) ->
+      .forEach { methodName, handlerMethod ->
```

Comment:
Seems like there was a problem using the inline version of `forEach` when one of the parameters is a suspend function.

Reply:
Good find!

Reply:
How did you ever figure this out???